### PR TITLE
Fix image path, delete temp directory

### PR DESCRIPTION
**Pull request by lucasmelogithub**

### CODE REVIEW

**Comment by JoshHilliker** [comment]
reviewed.  all changes look good to me..

**Comment by JoshHilliker** [comment]
Waiting for @mandalrajiv  approval to merge.

**Review by mandalrajiv — APPROVED** [review]

**Pull request merged**
